hbase-server/src/test/java/org/apache/hadoop/hbase/regionserver/TestMemStoreChunkPool.java

```diff
@@ -264,9 +272,151 @@ public void run() {
       t1.join();
       t2.join();
       t3.join();
-      assertTrue(newCreator.getPoolSize() <= maxCount);
+      assertTrue(exceptions[0] == null);
+      assertTrue(newCreator.getPoolSize() <= maxCount && newCreator.getPoolSize() > 0);
     } finally {
       ChunkCreator.instance = oldCreator;
     }
   }
+
+  @Test
+  public void testNullPointerExceptionWhenIndexChunksPollIsNullBug26142() throws Exception {
```

Comment:
Better give this method another name, like testNoIndexsPool or others, and add comment to say this is the UT for HBASE-26142, about NPE.

Reply:
@Apache9 ,Thank your for the review,  I have modified the code following your suggestion.